README.md

```diff
@@ -94,29 +94,10 @@ Once a major version enters LTS coverage, new features (semver-minor) may only
 be landed with consent of the Release working group. No semver-major
 changes other than those required for critical security fixes may be landed.
 
-Changes in an LTS-covered major version are limited to:
-
-1. Bug fixes;
-2. Security updates;
-3. Non-semver-major npm updates;
-4. Relevant documentation updates;
-5. Certain performance improvements where the risk of breaking existing
-   applications is minimal;
-6. Changes that introduce large amount of code churn where the risk of breaking
-   existing applications is low and where the change in question may
-   significantly ease the ability to backport future changes due to the
-   reduction in diff noise.
-
 Generally changes are expected to live in a *Current* release for at least 2
 weeks before being backported. It is possible for a commit to land earlier at
 the discretion of the Release working group and the maintainers of the LTS branches.
 
-Once a release moves into Maintenance mode, only ***critical*** bugs,
-***critical*** security fixes, documentation updates, and updates to ensure
-consistency and usability of the N-API across LTS releases will be permitted.
-Unless a change is ***urgent*** it will be planned into a release once per
-quarter. Such releases will only be made when necessary.
```

Comment:
This paragraph needs to stay -- It basically defines what being in `Maintenance` mode is. We have been following this for Node.js 6.x.

Node.js 8.x is an oddity in that according to the original schedule it shouldn't be in Maintenance until April but it was [brought forward to January](https://github.com/nodejs/Release/issues/376) at the end of last year. So far nothing in https://github.com/nodejs/Release/issues/405 has actually landed in an 8.x release and the things being discussed there are not guaranteed to land either (in fact this very paragraph, unless changed, should be the guidance as to what is accepted).